packages/client-engine-runtime/src/interpreter/renderQuery.test.ts

```diff
@@ -0,0 +1,178 @@
+import { PlaceholderFormat, QueryPlanDbQuery } from '../QueryPlan'
+import { renderQuery } from './renderQuery'
+import { ScopeBindings } from './scope'
+
+test('no template', () => {
+  expect(
+    renderQuery(
+      {
+        type: 'rawSql',
+        sql: 'SELECT * FROM users WHERE id = $1',
+        params: [1],
+      } as QueryPlanDbQuery,
```

Comment:
```suggestion
      } satisfies QueryPlanDbQuery,
```

Reply:
`as` asserts the type (even if it's not fully compatible) and can cause us to miss errors. `satisfies` actually checks it's assignable to the specified type.

Reply:
Thanks, I wasn't aware of this potential issue. I fix this in all places.